### ISSUE TITLE
Hi, Tried to do image analysis gor this error

### ISSUE THREAD

**Issue by omergassner-PM** [abi/screenshot-to-code#557]
installHook.js:1 Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.
overrideMethod @ installHook.js:1Understand this warning
installHook.js:1 `DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.

If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.

For more information, see https://radix-ui.com/primitives/docs/components/dialog
overrideMethod @ installHook.js:1Understand this error
installHook.js:1 Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.
overrideMethod @ installHook.js:1Understand this warning
console.js:36 Connecting to backend @  wss://backend-screenshot-to-code.onrender.com/generate-code
8An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.Understand this warning
installHook.js:1 Error generating code Please subscribe to a paid plan to generate code. If you are a subscriber and seeing this error, please contact support.
overrideMethod @ installHook.js:1Understand this error
console.js:36 Connection closed 1005 
installHook.js:1 Unknown server or connection error CloseEvent
